src/templates/Header/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint react/prop-types: 0 */
```

Comment:
wtf ? we agreed to not disable eslint for the entire file ? just add the proptypes for that component

Reply:
i resolved it after pr